File: java/dsframework/src/main/java/dsf/core/Message.java
package dsf.core;

/**
 *
 * @author arksea
 */
public class Message {

    public final String name;
    public final Object value;

    public Message(String name, Object value) {
        this.name = name;
        this.value = value;
    }

    @Override
    public String toString() {
        return name + ":" + value;
    }
}

File: csharp/dsf/dsf/src/core/Message.cs
namespace com.baidu.dsf.core
{

    /// 
    /// <summary>
    /// @author arksea
    /// </summary>
    public class Message
    {

        public readonly string name;
        public readonly object value;

        public Message(string name, object value)
        {
            this.name = name;
            this.value = value;
        }

        public override string ToString()
        {
            return name + ":" + value;
        }
    }

}